helm/alfresco-content-services/community_values.yaml

```diff
@@ -1,6 +1,7 @@
 # This values file can be used to install the latest ACS Community version using
 # the latest version of the chart
 alfresco-repository:
+  edition: Community
```

Comment:
AFAIR this is not used anymore and can be removed (was only used in the `NOTES.txt`).

Reply:
indeed there are no references in the charts, removing it